pkg/controller/common/helm/manifestprocessing.go

```diff
@@ -73,10 +73,14 @@ func (p *ManifestProcessor) ProcessManifest(ctx context.Context, man manifest.Ma
 			allErrors = append(allErrors, err)
 			continue
 		}
+		if len(rawJSON) == 0 || string(rawJSON) == "{}" || string(rawJSON) == "null" {
+			// MAISTRA-2227 ignore empty objects.  this could happen if charts have in empty blocks, e.g. superfluous "---"
+			continue
+		}
```

Comment:
Can we have a test for this?

Reply:
done

Reply:
I think the original error was not thrown in this function, was it? If my guess is true, then checking if err != nil in the test would not be enough, right?